aws-lc-rs-testing/benches/kem_benchmark.rs

```diff
@@ -0,0 +1,74 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+
+use aws_lc_rs::{
+    kem::DecapsulationKey,
+    unstable::kem::{get_algorithm, AlgorithmId},
+};
+use criterion::{criterion_group, criterion_main, Criterion};
+
+const UNSTABLE_ALGORITHMS: &[Option<&aws_lc_rs::kem::Algorithm<AlgorithmId>>] = &[
+    get_algorithm(AlgorithmId::Kyber512_R3),
+    get_algorithm(AlgorithmId::Kyber768_R3),
+    get_algorithm(AlgorithmId::Kyber1024_R3),
+];
+
+fn bench_kem_keygen(c: &mut Criterion) {
+    for ele in UNSTABLE_ALGORITHMS {
```

Comment:
just curious, what does `ele` stand for? :)

Reply:
Just a shortened variable name for the word "element".